src/_provisional.scss

```diff
@@ -149,6 +149,528 @@
+			top: 0;
+		}
+	}
+
+	@mixin gcweb-form-inline {
+		@media (min-width: $screen-sm-min) {
+			.form-group {
+				margin-bottom: 0;
+			}
+
+			.has-feedback .form-control-feedback {
+				top: 0;
+			}
+		}
+	}
+
+	.form-inline {
+		@include gcweb-form-inline;
+	}
+
+	.form-horizontal {
+		.radio,
+		.checkbox,
+		.radio-inline,
+		.checkbox-inline {
+			padding-top: ($gcweb-padding-base-vertical + 1);
+		}
+
+		.radio,
+		.checkbox {
+			min-height: ($gcweb-line-height-computed + ($gcweb-padding-base-vertical + 1));
+		}
+
+		@media (min-width: $screen-sm-min) {
+			.control-label {
+				padding-top: ($gcweb-padding-base-vertical + 1);
+			}
+		}
+
+		.form-group-lg {
+			@media (min-width: $screen-sm-min) {
+				.control-label {
+					font-size: $gcweb-font-size-large;
+					padding-top: ($gcweb-padding-large-vertical + 1);
+				}
+			}
+		}
+
+		.form-group-sm {
+			@media (min-width: $screen-sm-min) {
+				.control-label {
+					font-size: $gcweb-font-size-small;
+					padding-top: ($gcweb-padding-small-vertical + 1);
+				}
+			}
+		}
+	}
+
+
+
+	/*Input groups... (derived from https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_input-groups.scss)*/
+
+	.input-group-addon {
+		border-radius: $gcweb-input-border-radius;
+		font-size: $gcweb-font-size-base;
+		padding: $gcweb-padding-base-vertical $gcweb-padding-base-horizontal;
+
+		&.input-sm {
+			border-radius: $gcweb-input-border-radius-small;
+			font-size: $gcweb-font-size-small;
+			padding: $gcweb-padding-small-vertical $gcweb-padding-small-horizontal;
+		}
+
+		&.input-lg {
+			border-radius: $gcweb-input-border-radius-large;
+			font-size: $gcweb-font-size-large;
+			padding: $gcweb-padding-large-vertical $gcweb-padding-large-horizontal;
+		}
+	}
+
+	.input-group .form-control:first-child,
+	.input-group-addon:first-child,
+	.input-group-btn:first-child > .btn,
+	.input-group-btn:first-child > .btn-group > .btn,
+	.input-group-btn:first-child > .dropdown-toggle,
+	.input-group-btn:last-child > .btn:not(:last-child):not(.dropdown-toggle),
+	.input-group-btn:last-child > .btn-group:not(:last-child) > .btn {
+		@include border-right-radius(0);
+	}
+
+	.input-group .form-control:last-child,
+	.input-group-addon:last-child,
+	.input-group-btn:last-child > .btn,
+	.input-group-btn:last-child > .btn-group > .btn,
+	.input-group-btn:last-child > .dropdown-toggle,
+	.input-group-btn:first-child > .btn:not(:first-child),
+	.input-group-btn:first-child > .btn-group:not(:first-child) > .btn {
+		@include border-left-radius(0);
+	}
+}
+
+form {
+	&.provisional {
+		@extend %gcweb-main-font-size-line-height;
+		@extend %gcweb-form-font-new;
+	}
+}
+
+.provisional {
+	&.btn {
+		@extend %gcweb-btn;
+	}
+
+	&.btn-lg {
+		@extend %gcweb-btn-lg;
+	}
+
+	&.btn-sm {
+		@extend %gcweb-btn-sm;
+	}
+
+	&.btn-xs {
+		@extend %gcweb-btn-xs;
+	}
+
+	&.btn-call-to-action {
+		@extend %gcweb-btn-call-to-action-margin;
+	}
+
+	&.gc-form-font-new {
```

Comment:
It was decided that the class name required to get the increased font size in forms would temporarily be `.gc-font-2019` until it becomes stable, and will not be used in pair with the ".provisional" class.

Reply:
Ok cool :). I'll update this PR to match in my next set of updates.

Btw is the ``gc-font-2019`` class meant to be used in any situation where the font size needs to be increased? For example, would all of the following coding samples be considered ok to use that class in?
* ``<div class="gc-font-2019">...</div>`` (or any other container element)
* ``<form class="gc-font-2019">...</form>``
* ``<a class="btn btn-default gc-font-2019">...</a>`` (for one-off-buttons... buttons with a parent element that uses ``gc-font-2019`` wouldn't need this)

IIRC the last time I discussed it with @duboisp a couple of months ago, his stance was that he wanted to scrap the dedicated class and make it possible to just throw the ``provisional`` class onto any ``<form>`` element or in combination with the ``btn`` class for one-off buttons. But as you can see in my notes, I had some concerns since I felt like there were still valid use cases where a non-``<form>`` container element might need to be used.